transforms/helpers/transform-helper.ts

```diff
@@ -249,14 +256,14 @@ function convertIdentifierActionToMethod(
     ),
   ]);
   const expr = j.functionExpression(null, [], returnBlock);
-
-  return withDecorators(
-    withComments(
-      j.classMethod('method', idAction.key, expr.params, expr.body),
-      idAction
-    ),
-    decorators
-  );
+  return j.classMethod.from({
```

Comment:
`j` is a pretty short variable name. Is it idiomatic in the codeshift community? If so, 👍

Reply:
It shows up once in the jscodeshift docs, but was consistent in this repo when I started, so I kept it.